### ISSUE TITLE
usage de OTGUI_API

### ISSUE THREAD

**Issue by adumasphi** [persalys/persalys#13]
In GitLab by @schueller on Nov 4, 2016, 14:50

les classes de otguiview ne sont pas exportées ModelEvaluationWizard, c'est utile si on veut faire plus de tests de gui cf !44 
```
$ grep -nr '^class' lib/src/|grep hxx|grep -v OTGUI_API
lib/src/base/otgui/Observer.hxx:28:class Observable;
lib/src/view/otgui/SobolResultWindowWidget.hxx:31:class SobolResultWindowWidget : public QScrollArea
lib/src/view/otgui/ProbabilisticModelWindow.hxx:44:class ProbabilisticModelWindow : public OTguiSubWindow
lib/src/view/otgui/OutputsSelectionGroupBox.hxx:31:class OutputsSelectionGroupBox : public QGroupBox
lib/src/view/otgui/FunctionalChaosResultWindow.hxx:29:class FunctionalChaosResultWindow : public ResultWindow
lib/src/view/otgui/ImportDesignOfExperimentPage.hxx:29:class ImportDesignOfExperimentPage : public ImportDataPage
lib/src/view/otgui/MomentsEstimatesTableGroupBox.hxx:30:class MomentsEstimatesTableGroupBox : public QGroupBox
lib/src/view/otgui/SensitivityAnalysisWizard.hxx:34:class SensitivityAnalysisWizard : public AnalysisWizard
lib/src/view/otgui/DataAnalysisWindow.hxx:33:class DataAnalysisWindow : public ResultWindow
lib/src/view/otgui/MinMaxTableGroupBox.hxx:30:class MinMaxTableGroupBox : public QGroupBox
lib/src/view/otgui/OTguiMenuBar.hxx:27:class OTguiMenuBar : public QMenuBar
lib/src/view/otgui/PythonPhysicalModelWindow.hxx:33:class CodeView : public QTableView
lib/src/view/otgui/PythonPhysicalModelWindow.hxx:42:class PythonPhysicalModelWindow : public OTguiSubWindow
lib/src/view/otgui/OTguiStatusBar.hxx:28:class OTguiStatusBar : public QStatusBar
lib/src/view/otgui/SobolResultWindow.hxx:29:class SobolResultWindow : public ResultWindow
lib/src/view/otgui/TaylorExpansionMomentsResultWindow.hxx:31:class TaylorExpansionMomentsResultWindow : public ResultWindow
lib/src/view/otgui/MonteCarloResultWindow.hxx:27:class MonteCarloResultWindow : public DataAnalysisWindow
lib/src/view/otgui/YACSPhysicalModelWindow.hxx:33:class YACSPhysicalModelWindow : public OTguiSubWindow
lib/src/view/otgui/ModelEvaluationResultWindow.hxx:29:class ModelEvaluationResultWindow : public ResultWindow
lib/src/view/otgui/StopCriteriaGroupBox.hxx:32:class StopCriteriaGroupBox : public QGroupBox
lib/src/view/otgui/MetaModelWindow.hxx:28:class MetaModelWindow : public OTguiSubWindow
lib/src/view/otgui/DataModelWindow.hxx:29:class DataModelWindow : public OTguiSubWindow
lib/src/view/otgui/PhysicalModelWindowWidget.hxx:35:class PhysicalModelWindowWidget : public QWidget
lib/src/view/otgui/ProbabilisticDesignPage.hxx:29:class ProbabilisticDesignPage : public QWizardPage
lib/src/view/otgui/LimitStateWindow.hxx:31:class LimitStateWindow : public OTguiSubWindow
lib/src/view/otgui/ModelEvaluationWizard.hxx:29:class ModelEvaluationWizard : public AnalysisWizard
lib/src/view/otgui/DesignOfExperimentWizard.hxx:31:class IntroPage : public QWizardPage
lib/src/view/otgui/DesignOfExperimentWizard.hxx:47:class DesignOfExperimentWizard : public OTguiWizard
lib/src/view/otgui/ParametersWidget.hxx:27:class ParametersWidget : public QWidget
lib/src/view/otgui/OTguiWizard.hxx:29:class OTguiWizard : public QWizard
lib/src/view/otgui/OTguiSubWindow.hxx:30:class OTguiSubWindow : public QMdiSubWindow
lib/src/view/otgui/ReliabilityAnalysisWizard.hxx:33:class ReliabilityAnalysisWizard : public AnalysisWizard
lib/src/view/otgui/AnalyticalPhysicalModelWindow.hxx:29:class AnalyticalPhysicalModelWindow : public OTguiSubWindow
lib/src/view/otgui/ImportDataModelPage.hxx:28:class ImportDataModelPage : public ImportDataPage
lib/src/view/otgui/MonteCarloReliabilityResultWindow.hxx:29:class MonteCarloReliabilityResultWindow : public ResultWindow
lib/src/view/otgui/ResultWindow.hxx:29:class ResultWindow : public OTguiSubWindow
lib/src/view/otgui/StudyTreeView.hxx:31:class StudyTreeView : public QTreeView
lib/src/view/otgui/SRCResultWindow.hxx:32:class SRCResultWindow : public ResultWindow
lib/src/view/otgui/CentralTendencyWizard.hxx:34:class CentralTendencyWizard : public AnalysisWizard
lib/src/view/otgui/OTguiToolBar.hxx:27:class OTguiToolBar : public QToolBar
lib/src/view/otgui/DataAnalysisResultWindow.hxx:27:class DataAnalysisResultWindow : public DataAnalysisWindow
lib/src/view/otgui/OTguiMdiArea.hxx:29:class OTguiMdiArea : public QMdiArea
lib/src/view/otgui/AnalysisExecutionFailedWindow.hxx:28:class AnalysisExecutionFailedWindow : public OTguiSubWindow
lib/src/view/otgui/ImportDataPage.hxx:33:class ImportDataPage : public QWizardPage
lib/src/view/otgui/DesignOfExperimentWindow.hxx:34:class DesignOfExperimentWindow : public OTguiSubWindow
lib/src/view/otgui/BlockSizeGroupBox.hxx:30:class BlockSizeGroupBox : public QGroupBox
lib/src/view/otgui/MetaModelAnalysisWizard.hxx:33:class MetaModelAnalysisWizard : public AnalysisWizard
lib/src/view/otgui/DeterministicDesignPage.hxx:31:class DeterministicDesignPage : public QWizardPage
lib/src/view/otgui/AnalysisWizard.hxx:28:class AnalysisWizard : public OTguiWizard
lib/src/view/otgui/OTStudyWindow.hxx:28:class OTStudyWindow : public OTguiSubWindow
lib/src/view/otgui/DataModelWizard.hxx:30:class DataModelWizard : public OTguiWizard
lib/src/view/utils/otgui/CollapsibleGroupBox.hxx:30:class CollapsibleGroupBoxPrivate;
lib/src/view/plot/otgui/CustomScaleEngine.hxx:28:class CustomScaleEngine : public QwtLinearScaleEngine
```

**Comment by adumasphi** [comment]
In GitLab by @schueller on Nov 7, 2016, 09:34

Mentioned in merge request !45

**Comment by adumasphi** [comment]
In GitLab by @schueller on Nov 7, 2016, 10:07

Status changed to closed by commit 6edc91a94c95dfb33dce97fcd69941605e3a37f0
